src/main/src/main/java/org/geoserver/template/TemplateUtils.java

```diff
@@ -18,6 +21,9 @@
  * @author Kevin Smith, Boundless
  */
 public class TemplateUtils {
+    /** Reference feature version for Freemarker templates */
+    public static Version GS_VERSION = Configuration.VERSION_2_3_0;
```

Comment:
Its in the noise, but GS_VERSION seems a bit strange. FM_VERSION or FREEMARKER_CONFIG_VERSION?

Reply:
FM_VERSION it is, updated.